shared/fs/footer/upload-countdown-hoc.tsx

```diff
@@ -1,71 +1,72 @@
-// @flow
 import * as Flow from '../../util/flow'
 import * as React from 'react'
 import {formatDuration} from '../../util/timestamp'
-import {HOCTimers, type PropsWithTimer} from '../../common-adapters'
-import {type UploadProps} from './upload'
+import {HOCTimers, PropsWithTimer} from '../../common-adapters'
+import {UploadProps} from './upload'
 
 export type UploadCountdownHOCProps = {
-  endEstimate?: number,
-  files: number,
-  fileName: ?string,
-  totalSyncingBytes: number,
-  debugToggleShow?: () => void,
+  endEstimate?: number
+  files: number
+  fileName: string | null
+  totalSyncingBytes: number
+  debugToggleShow?: () => void
 }
 
 type Props = PropsWithTimer<UploadCountdownHOCProps>
 
 // Cosider this component as a state machine with following four states. 1Hz
 // Ticks (from _tick() calls by setInterval) and props changes (through
 // componentDidUpdate() calls) are two possible inputs.
-type Mode =
+enum Mode {
   // The upload banner isn't shown.
-  | 'hidden'
+  Hidden = 1,
```

Comment:
why `= 1`?

Reply:
Not necessary; removing. I was (incorrectly) under the impression this was needed for it to be integer underneath. Just realized it just defaults to start from 0 if not assigned a thing.